controllers/profiles/common/test/common.go

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package common_test
+
+import (
+	"context"
+
+	"github.com/apache/incubator-kie-kogito-serverless-operator/controllers/discovery"
+)
+
+const (
```

Comment:
This file was originally generated to leave in the context of the properties tests package, why do we need to move it outside it?

The question is because as we move to a common package the trend is that it'll degenerate and become a sort of all in one where people start putting the stuff their need for whatever test, etc.

Reply:
I also didn't like this move but it was needed because, in one of the commits (long history), those functions and constants were used by test functions in different packages, which created a sort of circular dependency. maybe now I can put it back into the original package, let me investigate. Thanks for raising this

Reply:
Good catch @wmedvede: I did not realize that some of the recent rebases have restored the original code, so that `common_test` package was not used at all, so I just removed it. Thanks!